src/views/ChatView.vue

```diff
@@ -69,15 +86,53 @@ export default {
     QToolbar,
     QToolbarTitle,
     QBtn,
-    QInput
+    QPopover,
+    QInput,
+    QToggle,
+    QList,
+    QItem,
+    QItemMain,
+    QOptionGroup,
+    Alert
   },
   methods: {
     exitChat() {
       this.$router.push({ path: '/' });
+    },
+    anom(anonymous) {
+      let alert = null;
+      if (anonymous) {
+        document.getElementsByClassName('q-toggle-handle')[0].style = 'background-color: rgb(27, 188, 155); border: none;';
```

Comment:
I don't recommend this. Consider, at least, toggling a classname on the element, where the existence of that classname modifies its styling.

Reply:
Agreed

Reply:
I don't get this sorry. Can explain further?